src/store/menu/__tests__/menu.reducers.test.ts

```diff
@@ -0,0 +1,20 @@
+import { menuReducer } from 'store/menu/reducers/menu.reducer';
+import { initialState } from '../hardcoded-data';
+
+describe('test menu reducer', () => {
+  it('should handle @menu/off', () => {
+    expect(
+      menuReducer(initialState, {
+        type: '@menu/off',
+      }),
```

Comment:
As far as I know we used `@typesafe-actions` to create actions, so, is there any solution to create actions in unit tests by using this library? Instead of creating it manually. We could re-use already defined action creators

Reply:
good point, changed